amethyst_core/src/lib.rs

```diff
@@ -55,6 +53,7 @@ mod hide_system;
 mod named;
 mod system_desc;
 mod system_ext;
+mod thread_pool;
 
-/// A rayon thread pool wrapped in an `Arc`. This should be used as resource in `World`.
-pub type ArcThreadPool = Arc<rayon::ThreadPool>;
+/// A rayon thread pool proxy. This should be used as resource in `World`.
+pub type ThreadPool = thread_pool::ThreadPool;
```

Comment:
Could just as well call it `ArcThreadPool` as it just wraps the `Arc<rayon::ThreadPool>` but the user is not expected to clone it so this may fit better.